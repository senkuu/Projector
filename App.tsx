/**
 * Sample React Native App
 * https://github.com/facebook/react-native
 *
 * Generated with the TypeScript template
 * https://github.com/react-native-community/react-native-template-typescript
 *
 * @format
 */

import React from 'react';
import Main from 'views/Main';
import { View } from 'react-native';

const App: React.FC = () => {
  return (
    <View style={{ backgroundColor: '#f2f2f2' }}>
      <Main />
    </View>
  );
};

export default App;
