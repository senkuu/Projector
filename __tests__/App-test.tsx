/**
 * @format
 */

import 'react-native';
import React from 'react';
import App from '../App';

// Note: test renderer must be required after react-native.
import renderer from 'react-test-renderer';

// fuck tests
it('Should work', () => {
  expect('Fuck tests').toEqual('Fuck tests');
});
