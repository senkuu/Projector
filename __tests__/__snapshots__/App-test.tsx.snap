// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View>
  <View>
    <Text
      style={
        Object {
          "fontFamily": "Montserrat",
          "fontSize": 24,
          "fontWeight": "bold",
        }
      }
    >
      Projector
    </Text>
  </View>
</View>
`;
